src/components/FormFields.tsx

```diff
@@ -36,42 +41,51 @@ export const TextField = createRestyleComponent<Props, Theme>(
   TextInput,
 )
 
-type LabeledTextField = {
+type FormFieldProps = BoxProps<Theme> & {
   label: string
-  value: string
+  value: any
   onChange?: (string) => void
   onTap?: () => void
+  labelProps?: TextProps<Theme>
   disabled?: boolean
 }
 
-export const LabeledTextField: React.FC<LabeledTextField> = ({
+export const LabeledTextField: React.FC<FormFieldProps> = ({
   label,
   value,
   onChange,
   onTap,
   disabled,
+  labelProps = {},
   ...props
 }) => (
-  <Box width="100%" py={4}>
-    <Text fontWeight="bold" color="darkGrey" pb={2}>
-      {label}
-    </Text>
+  <Box width="100%" py={4} {...props}>
+    {!!label && (
```

Comment:
Nice! I haven't managed to use this operator before

Reply:
Yeah it's useful for the `{conditional &&  (Component)}` style because if the conditional variable is an empty string then you will get a rendering error because it is conditionally false but will render an empty string because `'' && <Text>...</Text> = ''`.